runtime/libpgmath/lib/common/fltfenv.c

```diff
@@ -507,7 +507,7 @@ __fenv_feraiseexcept(int exc)
 {
   return feraiseexcept(exc);
 }
-
+#if !defined(TARGET_WIN_ARM64)
```

Comment:
Where will these functions come from in Windows?

Reply:
> @tdusnoki will you be updating the patch and replying to the remaining comments?

Yes its currently WIP, I'll update it asap.

Reply:
This needs to be implemented on Windows as these functions are not included in fenv.h. Currently I'm testing possible workarounds and implementations for it.